hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/test/java/org/apache/hadoop/yarn/api/TestPBImplRecords.java

```diff
@@ -430,6 +431,7 @@ public static void setup() throws Exception {
     generateByNewInstance(UpdatedContainer.class);
     generateByNewInstance(ContainerUpdateRequest.class);
     generateByNewInstance(ContainerUpdateResponse.class);
+    generateByNewInstance(EnhancedHeadroom.class);
```

Comment:
No other tests?

Reply:
Since this PR is just handling proto side of changes, will be adding more UT when using EnhancedHeadroom info for calculating cluster load.

Reply:
I meant more related to check that the fields are there for example.